perma_web/api/tests/test_link_authorization.py

```diff
@@ -168,10 +168,10 @@ def test_should_allow_user_to_patch_with_file(self):
 
         self.assertTrue(Capture.objects.filter(link_id=self.link.pk, role='primary').exclude(pk=old_primary_capture.pk).exists())
 
-        # This link now only has a warc, but not a wacz
+        # This link still only has a wacz
         self.link.refresh_from_db()
-        self.assertTrue(self.link.warc_size)
-        self.assertFalse(self.link.wacz_size)
+        self.assertFalse(self.link.warc_size)
+        self.assertTrue(self.link.wacz_size)
```

Comment:
I think this test needs further changes, with this new setup, to prove that the patch worked. Before, demonstrating that once there was a WACZ and now there is a WARC demonstrated that the process succeeded. Now, just asserting that warc_size is still zero and wacz_size is still non-zero doesn't, I think, assert much.

Instead, I think it would be more meaningful to a) check for the presence of a new `Capture` object with `role='provenance_summary'` (more on that below) and b) assert that the wacz_size is different than what it was before. That still doesn't assert anything about the contents of the WACZ (easier to test an upload-your-own playback in our functional tests), but I think that might be more complete?

Reply:
Yeah, sounds good -- let me see about adding and checking a new `Capture` object.